website/client-old/js/services/userServices.js

```diff
@@ -190,6 +190,8 @@ angular.module('habitrpg')
         })
       }
 
+      //Calling this multiple times rapidly will not persist changes
```

Comment:
That's strange.. it's possible that between one call and the other the server responded with the updated user that then overwrote the second call?